### ISSUE TITLE
Option to change scale/zoom for maps in embedded reports. 

### ISSUE THREAD

**Issue by improwise** [matomo-org/matomo#15794]
Is it possible to display maps of realtime users etc. with some default zoom or similar? Almost all of our visitors are from a small part of the world so always seeing the world map by default makes all dots look stacked on each other. It seems that sometimes the reports are displayed with zoom, sometimes not? Or do the remember the last setting of the last user?

**Comment by sgiehl** [comment]
If I remember correctly the maps automatically zooms in if all visitors were from the same continent or something like that

**Comment by tsteur** [comment]
@improwise we're already indeed zooming in should all visit be from one country for example. Do you maybe have visitors from different countries?

Remembering the last selection be indeed ideal. It would be ignored should all visits be from the same country.

**Comment by improwise** [comment]
Yes, the could be one or two from neighboring countries, as for most sites I would imagine. Even if you target just one country, people go on vacations etc. Just being able to default zoom to a continent would be a good start.
